NEON_plant_phenology.Rmd

```diff
@@ -590,3 +458,142 @@ By examining these two species side by side, we can see that *Lonicera maackii*
 peaks at about the same time as *Juglans nigra*, but it gets leaves earlier 
 and keeps its leaves later. To actually distinguish each of these events, 
 we would need a more complicated model with more fit parameters.
+
+When phenologists model phenophase changes, they often use *growing degree
+days* (GDD) instead of the true date, since plants respond to environmental signals
+like temperature and not to the calendar date. In the extension below
+we will calculate cumulative growing degree days and plot the phenophase 
+intensity as above vs. cumulative growing degree days. 
+
+## Extension - Growing Degree Days
+
+There are several methods to calculate degree days [@Battel2017; @Nugent2004; @Miller2018]. 
+Here we will use a simple method calculated by averaging the maximum and minimum 
+temperatures in a day and subtracting a base temperature below which no growth is expected:
+
+$$\text{GDD} = \frac{\text{max temp} - \text{min temp}}{2} - \text{base temp}$$ 
+
+Negative GDD values are set to zero. To choose a base temperature, we ideally need to know 
+how the species we're investigating grows. For illustration purposes here, we'll choose
+a base temperature of 10 degrees Celsius (50 degrees Fahrenheit).
+
+To combine this information with the phenology information above, we will use air temperature 
+measurements collected every 30 minutes at two of the NEON sites we're looking at.
+
+```{r, message = FALSE}
+# 30 minute averages of 1Hz observations of air temperature in degrees Celsius
+temp <- read_csv('NEON-pheno-temp-timeseries/temp/SAAT_30min.csv') 
+
+# what are the column names?
+names(temp)
+```
+
+First, let's convert the date and time column to a date column named `date`, and the temperature measurements to a numeric column. 
+
+```{r}
+temp$date <- as.Date(temp$startDateTime)
+
+temp$tempSingleMean <- as.numeric(temp$tempSingleMean)
+```
```

Comment:
Not strictly necessary at all, but it's possible to tidyverse-ify this like so:

```R
temp <- temp %>%
    mutate(date = as.Date(startDateTime),
           tempSingleMean = as.numeric(tempSingleMean))
```

Might help with consistency - although it could be equally useful for students to know of this alternate method. Your call!